hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutManagement.java

```diff
@@ -435,29 +432,14 @@ private Long assignTargetsToGroupInNewTransaction(final JpaRollout rollout, fina
 
             createAssignmentOfTargetsToGroup(targets, group);
 
-            return Long.valueOf(targets.getNumberOfElements());
+            return (long) targets.getNumberOfElements();
         });
     }
 
     private void createAssignmentOfTargetsToGroup(final Page<Target> targets, final RolloutGroup group) {
         targets.forEach(target -> rolloutTargetGroupRepository.save(new RolloutTargetGroup(group, target)));
     }
 
-    @Override
```

Comment:
@Reviewer: this overrides parent with exactly the same implementation, is this intended?